meta-ostro/classes/ostro-image.bbclass

```diff
@@ -184,7 +184,8 @@ FEATURE_PACKAGES_ima = "packagegroup-ima-evm-utils"
 FEATURE_PACKAGES_iotivity = "packagegroup-iotivity"
 FEATURE_PACKAGES_devkit = "packagegroup-devkit \
     ${@ bb.utils.contains('IMAGE_FEATURES', 'java-jdk', 'mraa-java upm-java', '', d)} \
-    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa', '', d)} \
+    ${@ bb.utils.contains('IMAGE_FEATURES', 'python-runtime', 'python-mraa python-upm', '', d)} \
```

Comment:
Does this python-upm depend on some config change? I was not getting such a package when I worked on https://github.com/ostroproject/meta-ostro/pull/219/commits/1f3eb822d9259e122b7bee8eddf990db07723e59, that's why it wasn't added earlier. I thought it was intentional that there is no Python binding for UPM.